modules/cache-common/src/main/java/org/opensearch/common/cache/stats/TieredSpilloverCacheStatsHolder.java

```diff
@@ -6,9 +6,9 @@
  * compatible open source license.
  */
 
-package org.opensearch.cache.common.tier;
+package org.opensearch.common.cache.stats;
 
-import org.opensearch.common.cache.stats.DefaultCacheStatsHolder;
+import org.opensearch.common.cache.tier.TieredSpilloverCache;
```

Comment:
Why do we need to include TSC class here?

Reply:
Think something got automatically imported when I had refactored the packages since they ended up in different packages; now that I've reverted this change this import isn't needed anymore.